canonical-websites/cn.ubuntu.com
# remove phone and tablet and update homepage

## Done

 - Removed phone and tablet sections
 - Removed refs to phone and tablet on homepage
 - Updated copy and one image on homepage for 17.04

## QA

Run site, check that general navigation works and that the homepage matches the [copydoc](https://docs.google.com/document/d/1jk_uZ0Qd2EuyqHAHxrpvK07QIagilGJdXEV_iUtP6qo/edit)

[Demo](http://cn.ubuntu.com-17-04-homepage-updates.demo.haus/)
